readme.md

```diff
@@ -40,6 +40,7 @@ You can take all the files of this site and run them just on your computer as if
 * [Jekyll](https://jekyllrb.com/)
 * [Ruby](https://www.ruby-lang.org/en/)
 * [Git](https://git-scm.com/)
+* [A JavaScript runtime (ie. node.js)](https://github.com/rails/execjs)
```

Comment:
I think it's confusing to link to execjs which is not used.

Reply:
@mlinksva FYI here's an example of execjs being referenced in the error output from jekyll:

- https://github.com/jekyll/jekyll/issues/2327#issue-32978237

(see first line of stacktrace)

The issue goes on to describe a dependency on coffeescript which is going to be deprecated in jekyll V3, and it looks like in [some cases](https://github.com/jekyll/jekyll/issues/2327#issuecomment-54740996) nodejs was the only solution that worked.

The error message is also noted here:

jekyll docs: 

- https://github.com/jekyll/jekyll/blob/master/docs/_docs/troubleshooting.md#could-not-find-a-javascript-runtime-execjsruntimeunavailable